packages/eslint-config/jest.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    overrides: [
+        {
+            files: ['*.spec.js', '*.test.js'],
+            env: {
+                jest: true,
+            },
+            extends: ['plugin:jest/recommended'],
+            plugins: ['jest'],
```

Comment:
This requires us to have the `eslint-plugin-jest` dependency as well. And add it in the readme as a requirement alongside the other dependencies...